.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: java
 jdk:
   - oraclejdk8
-  - oraclejdk7
```

Comment:
Why dropping Oracle Java7?

Reply:
@masci, please take a look at https://github.com/travis-ci/travis-ci/issues/7884
basically, oraclejdk7 is no longer available

Reply:
Thanks for the link, merging!